tests/conftest.py

```diff
@@ -201,11 +201,11 @@ def fuzzy_assert_proportion(
         if reject_null:
             if observed_proportion < target_lower_bound:
                 raise AssertionError(
-                    f"{name} value {observed_proportion:g} is significantly less than expected, bayes factor = {bayes_factor:g}"
+                    f"{name} value {observed_proportion:g} is significantly less than expected {target_lower_bound:g}, bayes factor = {bayes_factor:g}"
```

Comment:
what are the `:g` doing?

Reply:
I think it stands for "general" -- basically it puts the number in scientific notation if it is very large or very small and would require a lot of digits. This is something I want to improve in the future -- Patrick suggested some ways to do so.

Reply:
From the docs since I had to look it up to:
The precise rules are as follows: suppose that the result formatted with presentation type 'e' and precision p-1 would have exponent exp. Then, if m <= exp < p, where m is -4 for floats and -6 for [Decimals](https://docs.python.org/3/library/decimal.html#decimal.Decimal), the number is formatted with presentation type 'f' and precision p-1-exp. Otherwise, the number is formatted with presentation type 'e' and precision p-1. In both cases insignificant trailing zeros are removed from the significand, and the decimal point is also removed if there are no remaining digits following it, unless the '#' option is used.